### DISCUSSION TITLE
[Keyball61]特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by zaki-yama** [Yowkees/keyball#468] [Q&A] [answered]
Keyball61を購入し組み立てを行ったのですが、画像赤丸で囲った、左手のzキーに相当するキーが反応しません。

![IMG_8963](https://github.com/Yowkees/keyball/assets/1001444/77115e70-2356-4ef2-950f-445255b854b3)

元々、キースイッチは反応しないが [3-7．ProMicroの書き込みと動作確認
](https://github.com/Yowkees/keyball/blob/main/keyball61/doc/rev1/buildguide_jp.md#3-7promicro%E3%81%AE%E6%9B%B8%E3%81%8D%E8%BE%BC%E3%81%BF%E3%81%A8%E5%8B%95%E4%BD%9C%E7%A2%BA%E8%AA%8D)　に記載の方法でピンセットによる導通確認をしたところ反応がありました。
そのためキースイッチソケットの問題かと思い交換したのですが、問題は解決せず、今度はピンセットでも動作しなくなりました。

原因として考えられる点はございますでしょうか？

また、基盤図をダウンロードしKiCadで閲覧したのですが対応するダイオードが見つけられず、ご教示いただけますと幸いです。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

問題のキーに対応するダイオードは下記画像の緑丸で示したものになります。
このダイオードの両端子を半田ごてで触ってみて、脱落するようでしたらテンプラしていますので、半田付けしなおしてください。

また、キーソケットを交換する際に基板の配線が傷ついた可能性があります。
下記画像の青点どうし、オレンジ点どうしが導通しているかも再確認お願いします。もし導通していない場合には、エナメル線等でショートしていただければ回復できると思います。

以上、お手数ですがご対応お願いいたします。

![KB61_1keyFix](https://github.com/Yowkees/keyball/assets/84209772/1b6fe497-c1d3-4d09-9787-8ed64ec3821b)

**Reply by zaki-yama** [reply]
@Yowkees 
お返事が遅れてしまい申し訳ありません。
教えていただいた箇所を再度半田付けしたところ、反応するようになりました！

丁寧に教えてくださりありがとうございました。

**Comment by Yowkees** [comment]
ご対応ありがとうございます。
無事に動くようになったようで安心しました。
ぜひ末永くご愛用いただければ幸いです。
